static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -474,9 +496,9 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
 
       if (!isDetailCardOpened) {
         window.requestAnimationFrame(() => {
-          this._detailCard.style = 'height: 0;';
+          this._detailCard.setAttribute('style', 'height: 0;');
```

Comment:
do we need to worry about other usages of `.style` in the theme?

Reply:
The error happens only when you try to modify the entire style object like `element.style = 'height: 0;'` This is the only spot in the theme where we were modifying style like that.  Modifying it like `style.height = '300px'` or `style['height'] = '200px'` works fine in IE11. That's how we usually modify style in the theme

Reply:
I think the `setAttribute` call works here because it corresponds with `removeAttribute`